actionpack/lib/action_controller/metal/http_authentication.rb

```diff
@@ -429,14 +418,32 @@ module Token
       extend self
 
       module ControllerMethods
+        extend ActiveSupport::Concern
+
+        module ClassMethods
+          # Enables HTTP Token authentication.
+          #
+          # See ActionController::HttpAuthentication::Token for example usage.
+          def http_token_authenticate_with(token:, realm: nil, **options)
+            raise ArgumentError, "Expected token: to be a String, got #{name.class}" unless name.is_a?(String)
```

Comment:
This should be `token`, right?

```suggestion
            raise ArgumentError, "Expected token: to be a String, got #{token.class}" unless token.is_a?(String)
```